RawBencher/Program.cs

```diff
@@ -9,6 +8,13 @@
 using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
+using RawBencher.Benchers;
+using System.Threading;
+using Dapper;
+#if !DNXCORE50

```

Comment:
Where is DNXCORE50 defined? As it seems it's not defined here (2015 with b8) so the conditional compilations don't work.